contents/codes/308.md

```diff
@@ -16,14 +18,28 @@ A 308 response is cacheable by default; i.e., unless otherwise indicated by the
 
 Note: This status code is similar to [301 Moved Permanently](/301), except that it does not allow changing the request method from POST to GET.
 
+### Resume Incomplete
+
+This code indicates to the client that the server does not posses the
```

Comment:
Can you add this as the first sentence of this paragraph?

"Some applications may use `308 Resume Incomplete` as a non-standard response."